### DISCUSSION TITLE
Re-order Stations to make building a station easier while holding the book

### DISCUSSION THREAD

**Discussion by crackalackintv** [Opinionated-Blueprints/10-Books-Full-of-Rails#67] [Ideas]
I suggest changing the order of the book to something like

Stackers, Platforms, Providers, Requesters, Bufferless Providers, Inline P/R and then the depots and refueling, as that allows you to scroll through the book more easily to build a complete station than having to pick the sub-books individually / scroll through less-used stuff, like Depots, to get to Stackers.

**Comment by Bocian-1** [comment]
Is this really viable? There's so many blueprints I'd think scrolling through the entire Stations books would be even more of a pain?

**Reply by crackalackintv** [reply]
The biggest "issue" is that the stackers go through all the different train sizes, so I guess that is a lot of scrolling. 🤔 
Do you just always move the blueprint window to the side?

**Reply by Bocian-1** [reply]
That's what I gravitated towards in the end. I still close it all the time on accident though with 'e' and getting back to it is pretty annoying, but I kind of accepted it as a side effect of the sheer amount of everything

**Reply by crackalackintv** [reply]
Ok, I tried to do that as well now. Guess it is just too big of a book to scroll through. Will close this topic then.
